core/src/com/unciv/ui/newgamescreen/NewGameScreenOptionsTable.kt

```diff
@@ -36,55 +44,47 @@ class NewGameScreenOptionsTable(val newGameScreen: NewGameScreen, val updatePlay
         addBarbariansCheckbox()
         addOneCityChallengeCheckbox()
         addIsOnlineMultiplayerCheckbox()
-
-         addModCheckboxes()
+        addModCheckboxes()
 
         pack()
     }
 
     private fun addMapTypeSelection() {
         add("{Map type}:".toLabel())
         val mapTypes = arrayListOf("Generated")
-        if (MapSaver().getMaps().isNotEmpty()) mapTypes.add("Existing")
-
-        val mapFileLabel = "{Map file}:".toLabel()
-        val mapFileSelectBox = getMapFileSelectBox()
-        mapFileLabel.isVisible = false
-        mapFileSelectBox.isVisible = false
-
+        if (MapSaver().getMaps().isNotEmpty()) mapTypes.add(MapType.custom)
         val mapTypeSelectBox = TranslatedSelectBox(mapTypes, "Generated", CameraStageBaseScreen.skin)
 
-        val mapParameterTable = MapParametersTable(mapParameters)
+        val mapFileSelectBox = getMapFileSelectBox()
+        savedMapOptionsTable.defaults().pad(5f)
+        savedMapOptionsTable.add("{Map file}:".toLabel()).left()
+        // because SOME people gotta give the hugest names to their maps
```

Comment:
I thought this was your comment and I lol'd, and then I noticed this is one of mine 0_0

Reply:
_SOME_ comments shall stay forever. :rofl: